0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.      return loss / 2 * len(x_data_list)
</pre><p>I think there is a small error here. Shouldn't it be <code>loss / (2 * len(x_data_list))</code> ? What we have is <code>(loss x N) / 2</code>  rather than <code>loss / (2N)</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='170'/>

Reply:
<p>You are right, this was a bug. Thank you! </p>